data-structures/LinkedList.py

```diff
@@ -0,0 +1,121 @@
+class Node:
+    def __init__(self, value, next_node=None):
+        self.value = value
+        self.next = next_node
+
+
+class LinkedList:
+    def __init__(self):
+        self.head = None
+        self.total = 0
+
+    def add(self, value):
+        """ Add to the end of the list """
+        new_element = Node(value)
+        if not self.head:
+            self.head = new_element
+        else:
+            current = self.head
```

Comment:
Can we change 'current' to 'pointer'?

Reply:
We can but i feel like current makes sense as a way of tracking. If it bugs you i can change to pointer.

Reply:
Okay, I agree